oviewer/convert_es.go

```diff
@@ -24,6 +24,18 @@ const (
 	oscURL
 )
 
+// Finalbyte is a character outside the escape sequence.
```

Comment:
```suggestion
// FinalByte is a character outside the escape sequence.
```

I could suggest a linter for that